info/v2/container.go

```diff
@@ -216,6 +216,9 @@ type FsInfo struct {
 
 	// Labels associated with this filesystem.
 	Labels []string `json:"labels"`
+
+	// Number of available Inodes.
+	InodesFree uint64 `json:"inodes_free"`
```

Comment:
this is problematic with devicemapper where inodes_free is not knowable since it operates at a block level.  I think we should make this a pointer.